shared/chat/inbox/index.desktop.js

```diff
@@ -155,7 +155,7 @@ class Inbox extends PureComponent<Props, State> {
           <div style={_scrollableStyle} onScroll={this._onScroll}>
             <ReactList
               ref={this._setRef}
-              useTranslate3d={true}
+              useTranslate3d={false}
```

Comment:
i believe the 3d transforms are the cause of the weird clicking on rows being broken sometimes on desktop